cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -483,7 +483,9 @@ func NewControllerInitializers(loopMode ControllerLoopMode) map[string]InitFunc
 	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.DynamicResourceAllocation) {
 		register("resource-claim-controller", startResourceClaimController)
 	}
-
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.ValidatingAdmissionPolicy) {
+		register("validatingadmissionpolicy-controller", startValidatingAdmissionPolicyController)
```

Comment:
The name is a little confusing. The controller only handles the policy status update instead of the entire validatingadmissionpolicy feature

Reply:
Good point! We can rename it to `validatingadmissionpolicy-typechecking-controller` How confident are we that the type checking is the only part that needs to be in KCM?

Reply:
Type checking might not be the only part, but status update would be. I would suggest something like `validatingadmissionpolicy-status-controller` in case to add more info into status later

Reply:
Good point. Updating.